### ISSUE TITLE
Location of Pokemon in new update

### ISSUE THREAD

**Issue by MerlionRock** [PokemonGoF/PokemonGo-Bot#5630]
### Short Description
New Pogo update has location of where it is been caught. Question, the information is generated when the Pokemon spawned or after been caught?

### Possible solution
Investigation needed on this new update as it might affect users who snipe. If location of Pokemon is generated only after been caught, not an issue. But if location of Pokemon is generated when Pokemon spawned, it will be a big issue. 

You cannot be traveling in New York and Pokemon came from Tokyo, London, Etc right?

### How it would help others
Temporary Stop sniping function if Pokemon spawned with location.

**Comment by Anakin5** [comment]
There was already some infor regarding the location. How is this different than before ?

**Comment by MerlionRock** [comment]
@Anakin5 I don't know.... I just update Pokemon Go to 0.39 logged into one of the bot account, there are no location information for all the Pokemon caught....

Not a single Pokemon caught has the location capture information....

**Comment by Anakin5** [comment]
I though there was the cellid before

**Comment by Gobberwart** [comment]
This could be a killer for snipers. If the spawned pokemon has location(a,b) but you caught it in location(c,d) it could very easily be a red flag.

**Comment by MerlionRock** [comment]
@Gobberwart Yes, that's what I am trying to say. I have no idea how this  Pokemon Caught location works, so it's a query. The location of the Pokemon is generated at spawned or caught?

I don't snipe, so I have no idea.

This information could help snipers to determine if they want to continue to do so...

**Comment by Anakin5** [comment]
I just checked and Pokemon already had `captured_cell_id`  and `creation_time_ms` so it was already possible to know if you were snipping or not.

**Comment by Gobberwart** [comment]
No issue then @Anakin5 ?

**Comment by Anakin5** [comment]
Not more than before. They knew if you were snipping and they still know it now.

**Comment by Gobberwart** [comment]
Closing then. Snipe at your own risk.

**Comment by Anakin5** [comment]
For the details, captured_cell_id gives you an approximate position (~300m). So it is easy to see if you snipe, especially long distances.
With the new update, they might be able to detect more accurately snipe on short distances.

**Comment by MerlionRock** [comment]
@Anakin5 Thanks for the clarification.

So no Snipe for me. Luckily introduced the hunter method, at least bot won't wander aimlessly....

**Comment by prusswan** [comment]
Currently, the web ui does not display the caught location of the pokemon. That should be a separate issue?
